Swift/Tests/ReplicatorTest+Collection.swift

```diff
@@ -84,39 +84,38 @@ class ReplicatorTest_Collection: ReplicatorTest {
         
         var config = ReplicatorConfiguration(database: self.db, target: target)
         
-        let conflictResolver = TestConflictResolver({ (con: Conflict) -> Document? in
+        let conflictResolver: TestConflictResolver = TestConflictResolver({ (con: Conflict) -> Document? in
```

Comment:
I'm curious why it needs explicit type `: TestConflictResolver`?

Reply:
not necessary